tests/testthat/test-assignment_linter.R

```diff
@@ -107,4 +107,22 @@ test_that("arguments handle trailing assignment operators correctly", {
     ),
     assignment_linter(allow_trailing = FALSE)
   )
+
+  expect_lint(
+    "{x<-1\n#blah\ny<-2}",
```

Comment:
sorry again for the confusion, here's an example:

```r
expect_lint(
  trim_some("
    {
      x <- '#x'
      y <- '#y'
    }
  "),
  NULL,
  assignment_linter(allow_trailing = FALSE)
)
```

Reply:
Okay, now I really understand! Thanks for the clarification :)